.github/workflows/chart-release.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
       -
         name: Package Helm Chart
         run: |
-          sed -i "s/VERSION/${{ github.ref_name }}/g" ./${{ env.CHART_NAME }}/chart/Chart.yaml
+          sed -i -e '1 s/version: v[0-9]*\.[0-9]*\.[0-9]*/version: ${{ github.ref_name }}/; t' -e '1,// s//version: ${{ github.ref_name }}/' ./${{ env.CHART_NAME }}/chart/Chart.yaml
           helm package ./${{ env.CHART_NAME }}/chart -u -d ./helm-charts/
```

Comment:
Why not set `--appVersion` here instead of wrenching `sed` back and forth?

Reply:
because that would be way to simple. I will do that.